envFiles/.env.devcontainer

```
@@ -142,10 +140,22 @@ REDIS_TEST_MAPPED_PORT=6378
 ##### OTEL and Observability #####
-API_OTEL_ENABLED=true # make it "false" if don't want span logging
-API_OTEL_SERVICE_NAME="talawa-api"
-API_OTEL_ENVIRONMENT="local"
-API_OTEL_SAMPLING_RATIO="1"
-API_OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4318/v1/traces" #TODO: after integrating Observability tool
-
-API_SLOW_REQUEST_MS=500
\ No newline at end of file
+API_OTEL_ENABLED=true
+API_OTEL_ENVIRONMENT=local
+API_OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4318/v1/traces"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)